src/BraketSimulator.jl

```diff
@@ -283,4 +393,137 @@ function __init__()
         StateVectorSimulator{ComplexF64,StateVector{ComplexF64}}
 end
 
+@setup_workload begin
```

Comment:
Has this been moved here from simulator-v2 as part of the transition to Julia OQ parsing?

Reply:
What? I don't understand this question. This is a purely Julia block for precompilation, why would it be in the Python wrapper?

Reply:
Oh, i was just asking because this is newly introduced, but in the old world, I remember the circuits for pre compilation lived in the Python wrapper

Reply:
Yeah we moved all that here so you just precompile at first import, not every import